server/strategy.go

```diff
@@ -20,21 +20,29 @@ func DecideOps(c *cke.Cluster, cs *cke.ClusterStatus, resources []cke.ResourceDe
 
 	nf := NewNodeFilter(c, cs)
 
+	// 0. Skip this iteration, if Etcd is not available and one or more CP node(s) are not connected
```

Comment:
I think there is no risk to _start_ and _wait_ etcd members even some
nodes are not accessible via SSH.